plugins/logs/loggrepx_

```diff
@@ -40,12 +41,13 @@ Available config options include the following:
  env.[field]_critical - Critical level for specific logfile
 
 NOTE: for any variable with [field] in it, [field] is derived from the
-full logfile path by simply replacing all non-alphanumerics with
-underscores. For example, the "warning" field for the logfile
-\`/var/log/nginx/errors.log\` would be \`var_log_nginx_errors_log_warning\`
+full logfile path by simply removing the preceding slash and replacing
+all non-alphanumerics with underscores. For example, the "warning" field
+for the logfile \`/var/log/nginx/errors.log\` would be
```

Comment:
Could you verify please, that this looks reasonable in `perldoc FILENAME`? (see https://perldoc.perl.org/perlpod.html#Formatting-Codes)